api/pkg/cli/cmd/search/search.go

```diff
@@ -121,6 +122,11 @@ func (opts *options) run() error {
 }
 
 func (opts *options) validate() error {
+
+	if len(opts.Args) == 0 && len(opts.Kinds) == 0 && len(opts.Tags) == 0 {
```

Comment:
might be worth writing a helper utility `allEmpty(list ...[]string) bool` that returns true if all the `[]string` is empty, so that the code reads


```go
if allEmpty(opts.Args, opts.Kinds, opts.Tags) {
  err ...
}
```